i18n/others/en-US.json

```diff
@@ -41,6 +41,39 @@
   "Torpedo": "Torpedo",
   "AntiAir": "AntiAir",
   "Armor": "Armor",
-  "Luck": "Luck"
+  "Luck": "Luck",
+
+  "doyouknow-prefix": "#Tip# ",
+  "doyouknow-contents": [
+    "We frequently introduces new features in \"Tips\". You can disable it in settings though",
+    "We look forward to your suggestions! Please post it on our GitHub repo: poooi/poi",
+    "Hang round the settings and plugin list regularly. You may find new features!",
+    "Anti-catbomb (network retry) ALWAYS comes at risk. USE WITH CAUTIOUS!",
+    "poi never modifies or actively sends network packets, provided you use trusted poi and plugins",
```

Comment:
", provided you use trusted poi and plugins" -> ". Please get your Poi and plugins from trusted source." I guess?